### DISCUSSION TITLE
The space after code in an Issue shouldn’t be formatted as code

### DISCUSSION THREAD

**Discussion by bitcoin-tools** [community/community#112180] [Mobile] [answered]
Actual behavior - On the iOS App, when viewing an issue that starts with code formatting, the space after the code still shows up as code.

Expected behavior - the space after the code-formatted text should have normal formatting.

Here's a link to the issue: https://github.com/bitcoin-tools/nodebuilder/issues/390

![image](https://github.com/community/community/assets/156422466/4f29eb6f-a859-41f8-8142-8119711221a5)format

![image](https://github.com/community/community/assets/156422466/826b3c1e-6b1b-4614-b953-7eabfdebb0c3)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.152.0 (125147271)
Device: iPhone 15 Pro (iOS 17.3.1)
</details>

**Comment by taki-on** [comment, marked as answer]
Thank you for the bug report! I will file it in our bug backlog and we will triage it!
